system-parachains/collectives/collectives-polkadot/src/fellowship/mod.rs

```diff
@@ -244,3 +251,120 @@ impl pallet_salary::Config<FellowshipSalaryInstance> for Runtime {
 	// Total monthly salary budget.
 	type Budget = ConstU128<{ 100_000 * USDT_UNITS }>;
 }
+
+parameter_types! {
+	// TODO: reference the constant value from common crate when polkadot-sdk 1.5 is released.
+	pub const FellowshipTreasuryPalletId: PalletId = PalletId(*b"py/feltr");
+	pub const HundredPercent: Permill = Permill::from_percent(100);
+	pub const Burn: Permill = Permill::from_percent(0);
+	pub const MaxBalance: Balance = Balance::max_value();
+	// The asset's interior location for the paying account. This is the Fellowship Treasury
+	// pallet instance (which sits at index 65).
+	pub FellowshipTreasuryInteriorLocation: InteriorMultiLocation = PalletInstance(65).into();
```

Comment:
I won't block on this but it will be great if there is a way to not having hardcoded number here

Reply:
maybe we could have a test somewhere to assert the value matches

Reply:
This location is not actually the problem, can just read it directly (which I have changed it to do). The problem is more [here](https://github.com/polkadot-fellows/runtimes/blob/90f334bd2eb31055b3178b0acb6bd5133a7c7ec7/system-parachains/asset-hubs/asset-hub-polkadot/src/xcm_config.rs#L198). I had added two tests and a link to an issue to pull these important pallet indices into a common area.